playground/frontend/lib/pages/playground/states/playground_state.dart

```diff
@@ -156,6 +166,39 @@ class PlaygroundState with ChangeNotifier {
       status: RunCodeStatus.finished,
       output: _selectedExample!.outputs,
     );
+    _executionTime?.close();
     notifyListeners();
   }
+
+  StreamController<int> _createExecutionTimeStream() {
+    StreamController<int>? streamController;
+    Timer? timer;
+    Duration timerInterval = const Duration(milliseconds: kExecutionTimeUpdate);
+    int ms = 0;
+
+    void stopTimer() {
+      if (timer != null) {
+        timer?.cancel();
```

Comment:
Redundant question mark or null check.

Reply:
question mark is required, because timer is not local variable

Reply:
The question mark is just a shorthand null check, so this code compiles to:
```
if (timer != null) {
    if (timer != null) {
       timer.cancel();
    }
}
```